File: rust/src/models/online_method_config.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, Default, PartialEq, Deserialize, Serialize)]
pub struct OnlineMethodConfig {
    pub enabled: bool,
}

impl OnlineMethodConfig {
    pub fn new(enabled: bool) -> Self {
        Self { enabled }
    }
}

File: typescript/src/models/onlineMethodConfig.ts
// this file is @generated

export interface OnlineMethodConfig {
  enabled: boolean;
}

export const OnlineMethodConfigSerializer = {
  _fromJsonObject(object: any): OnlineMethodConfig {
    return {
      enabled: object["enabled"],
    };
  },

  _toJsonObject(self: OnlineMethodConfig): any {
    return {
      enabled: self.enabled,
    };
  },
};
